mainwindow.ui

```diff
@@ -52,7 +55,7 @@
             <string>Launch</string>
            </property>
            <property name="icon">
-            <iconset resource="qtresources.qrc">
+            <iconset theme="system-run" resource="qtresources.qrc">
```

Comment:
The web is an example of something which goes the other way on who picks icons: it's all up to the designers.

I think there are only accepted semantic icons for a very few common actions, like the floppy disk for save. Neither 'launch' nor 'run' (which are subtly different) are among them.